open-bot.yaml

```diff
@@ -19,7 +19,7 @@ rules:
       identifier: "ci-result"
       readd: true
       message: |-
-        The most important CI builds succeeded. Great work so far.
+        Thank you for your pull request! We’ll review it soon.
```

Comment:
I’m not really sure is useful when CI is OK but can add it back if you think we need it ;-)

> Thank you for your pull request! The most important CI builds succeeded, we’ll review the pull request soon.

Good enough?